### PR TITLE
Add avctl deployment configs to agents

### PR DESCRIPTION
**Pull request by 5A11**
## Proposed Changes

Add avctl deployment configs to agents

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Bug fix
- [ ] New agent
- [ ] New solution
- [ ] New application

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/uAgents/blob/main/CONTRIBUTING.md) guide 
- [x] Checks and tests pass locally

### CODE REVIEW

File: 1-uagents/finance/company-ticker-resolver-agent/.avctl/config.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```
Comment (5A11) [inline 1-uagents/finance/company-ticker-resolver-agent/.avctl/config.toml:2]: Why is this empty (similarly some of the below)? Are they not deployed?

File: 1-uagents/finance/finance-q&a-agent/.avctl/config.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```
Comment (5A11) [inline 1-uagents/finance/finance-q&a-agent/.avctl/config.toml:2]: Same as above

File: 1-uagents/geo/open-charge-map-agent/.avctl/config.toml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```
Comment (5A11) [inline 1-uagents/geo/open-charge-map-agent/.avctl/config.toml:2]: Same as above

File: 1-uagents/knowledge-base/google-gemini-agent/.avctl/config.toml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```
Comment (5A11) [inline 1-uagents/knowledge-base/google-gemini-agent/.avctl/config.toml:2]: Same as above

File: 1-uagents/travel/flights-retriever-agent/.avctl/config.toml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```
Comment (5A11) [inline 1-uagents/travel/flights-retriever-agent/.avctl/config.toml:2]: Same as above

File: 1-uagents/utility/website-validation-agent/.avctl/config.toml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```
Comment (5A11) [inline 1-uagents/utility/website-validation-agent/.avctl/config.toml:2]: Same as above

File: 1-uagents/finance/company-ticker-resolver-agent/.avctl/config.toml (hunk 1 above)
Comment (jrriehl) [reply 1-uagents/finance/company-ticker-resolver-agent/.avctl/config.toml:2]: That's right, not deployed yet.

**Review by jrriehl — APPROVED** [review]
LGTM

**Pull request merged**